File: src/main/java/ru/hse/restaurant/mappers/OrderDbModelToOrderDtoMapper.java
package ru.hse.restaurant.mappers;

import org.mapstruct.Mapper;
import org.mapstruct.MappingConstants;
import ru.hse.restaurant.dto.Ordering;

@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
public interface OrderDbModelToOrderDtoMapper {
    Ordering map(ru.hse.restaurant.data.repositories.Ordering orderDbModel);
}

File: src/main/kotlin/ru/hse/RestaurantManager/mappers/OrderDbModelToOrderDtoMapper.kt
package ru.hse.RestaurantManager.mappers

import org.mapstruct.Mapper
import org.mapstruct.MappingConstants
import ru.hse.RestaurantManager.dto.Ordering as OrderingDto
import ru.hse.RestaurantManager.data.repositories.Ordering as OrderingDb

@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
public interface OrderDbModelToOrderDtoMapper {

    fun map(orderDbModel : OrderingDb) : OrderingDto {

    }
}